First cut string extraction

samzx opened the pull request:
First pass to make sure integration with zendesk translation service is working properly.

This required to be merged before: https://github.com/zendesk/rosetta/pull/2975

## Description
We extracted some strings to test functionality. This is the first pass, will mainly go unnoticed to users. If the translation service works, we'll be able to pull the Japanese translations and import it into this repo.

## References
[Github Issue](https://github.com/zendesk/volunteer_portal/issues/167)

## CCs
@ovillegaszen 
@zendesk/volunteer 

## Risks (if any)
* low - affects strings

Commit:
First cut string extraction

First batch of string extractions to test rosetta string sweep

Commit:
First cut string extraction

First batch of string extractions to test rosetta string sweep

Commit:
Added translation compatibility with calendar

Commit:
Added translation compatibility with calendar

Commit:
update snapshot, becomes Consumer from using withNamespaces

Commit:
update snapshot, becomes Consumer from using withNamespaces

samzx commented on app/javascript/components/Calendar/__snapshots__/test.js.snap:
```diff
@@ -1,31 +1,7 @@
-</Layout>
+<Consumer>
+  <Component />
+</Consumer>
```
This probably happened when wrapping the calendar component with `withNamespaces`

Commit:
two string extraction

Commit:
two string extraction

Commit:
Adding screenshots to string keys and fixing two issues on keys

Commit:
Merge branch 'ovillegas/i18next__rosetta' of github.com:zendesk/volunteer_portal into ovillegas/i18next__rosetta

Commit:
Adding screenshots to string keys and fixing two issues on keys

ThuTrinh commented on app/javascript/components/Calendar/index.js:
```diff
@@ -21,7 +23,7 @@ const styles = {
 
 // Custom components given to BigCalendar
-const calendarComponents = (currentUser, offices, filters, filterActions) => {
+const calendarComponents = (currentUser, offices, filters, filterActions, t) => {
```
would be good practice to give `t` a more descriptive name for readability.

samzx replied:
I think it's standard to have `t` as the translation function ie. the API gives the example: https://www.i18next.com/overview/api

What do you think?

ThuTrinh replied:
yeah , I wasn't aware that this prop was added by a third-party. Ignore me :)

ThuTrinh commented on app/javascript/components/Calendar/index.js:
```diff
@@ -145,11 +148,18 @@ const Calendar = ({
-          changeEventFilter,
-          changeOfficeFilter,
-        })}
+        culture="en"
```
is this the culture default value or is it hard coded?

samzx replied:
we're just explicitly setting the culture here so in the future we can change it via an interface / menu. But I think "en" is the default

Commit:
Merge branch 'ovillegas/i18next__rosetta' of github.com:zendesk/volunteer_portal into ovillegas/i18next__rosetta

ThuTrinh commented on app/javascript/components/Toolbar/index.js:
```diff
@@ -94,33 +94,52 @@ const OfficeFilter = ({ value, onChange, offices }) => (
       <div className={s.filterDropdowns}>
-        <ShowFilter value={showFilter.value} onChange={(_e, _i, value) => showFilter.onChange(value)} />
-        <EventFilter value={eventFilter.value} onChange={(_e, _i, value) => eventFilter.onChange(value)} />
+        <ShowFilter t={t} value={showFilter.value} onChange={(_e, _i, value) => showFilter.onChange(value)} />
```
`t` as a component property is not very descriptive. This makes it hard to quickly obtain context or understand how to set this value.

Commit:
update snapshot to satisfy jest

Commit:
Changes to string keys to start with volunteer_portal
Replace hardcoded text with string keys for pages/admin/Dashboard/index.js
Creation of string keys for the file
Screenshots to be added in the yml in a future PR

Commit:
Merge branch 'ovillegas/i18next__rosetta' of github.com:zendesk/volunteer_portal into ovillegas/i18next__rosetta

Commit:
remove non-served compiled files
